src/frontend/komponenter/Fagsak/Vedtak/FeilutbetaltValuta/FeilutbetaltValuta.tsx

```diff
@@ -33,6 +33,12 @@ const FlexRowDiv = styled.div`
     justify-content: space-between;
 `;
 
+const KopierTilNøsKnapp = styled(CopyToClipboard)`
+    :not(:hover):not(:active) {
+        color: var(--navds-semantic-color-link);
```

Comment:
Som jeg skrev i beskrivelsen til PRen: vi bruker gammel versjon av ds-tokens og jeg må derfor bruke det gamle navnet på fargen. Vi burde nok oppgradere til nyeste versjon snart, det gjør det enklere å finne riktig farge!

Reply:
Hmm, du skal kunne hente den fra ds-tokens med det gamle navnet, altså NavdsSemanticColorLink

Reply:
Da blir det også tydeligere at denne verdien må rettes når vi oppgraderer ds-tokens og får nye navn på fargene - for den finner ikke lenger den gamle verdien

Reply:
Og - enig i at vi bør oppgradere 😄

Reply:
Endret nå! Men er flere steder i koden vi skriver fargen på denne måten, så må huske å sjekke det opp når vi oppgraderer 👍